template/src/ontology/Makefile.jinja2

```diff
@@ -915,7 +915,7 @@ $(ONT).owl: $(ONT)-{{ project.primary_release }}.owl
 {% endif -%}
 
 # ROBOT pipeline that merges imports, including components.
-ROBOT_RELEASE_IMPORT_MODE={% if project.use_edit_file_imports -%}$(ROBOT) merge --input $< {% else -%}
+ROBOT_RELEASE_IMPORT_MODE={% if project.use_edit_file_imports -%}$(ROBOT) merge --input $< {% if project.import_group is defined and project.import_group.use_base_merging is false and project.import_group.annotate_defined_by %}--annotate-defined-by true{% endif -%}{% else -%}
```

Comment:
Is this conditional needed? `and project.import_group.use_base_merging is false`? seems redundant. If you try to annotate an already annotated class, nothing happens!

Reply:
(but please test this if true) :D

Reply:
Ok! Looks good then